### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by artegoser**

### CODE REVIEW

File: packages/plugins/src/middlewares/highlight.tsx [hunk 1]
```diff
@@ -20,14 +20,12 @@ function Highlighter({ content }: { content: string }) {
...
-        @import
-        "https://cdn.jsdelivr.net/gh/highlightjs/cdn-release@11.9.0/build/styles/atom-one-dark.min.css"
-        screen and (prefers-color-scheme: dark);
+        @import "/static/third-party/styles/atom-one-light.min.css"; @import
+        "/static/third-party/styles/atom-one-dark.min.css" screen and
+        (prefers-color-scheme: dark);
       </style>
       <script
-        src="https://cdn.jsdelivr.net/gh/highlightjs/cdn-release@11.9.0/build/highlight.min.js"
+        src="/static/third-party/scripts/highlight.min.js"
```
Comment (DarkCat09) [inline packages/plugins/src/middlewares/highlight.tsx:28]: 👍🏼

File: packages/server/templates/big_badge.ejs [hunk 2]
```diff
@@ -0,0 +1,52 @@
+<%
```
Comment (DarkCat09) [inline packages/server/templates/big_badge.ejs:1]: Add copyright notice, please. As I can see, this SVG code is copied from some badge.

Comment (artegoser) [reply packages/server/templates/big_badge.ejs:1]: I made this badge in Figma, then exported it to SVG.

Comment (artegoser) [reply packages/server/templates/big_badge.ejs:1]: we discussed the creation

Comment (DarkCat09) [reply packages/server/templates/big_badge.ejs:1]: Ah, sorry, forgot about that

**Review by DarkCat09 — APPROVED** [review]
LGTM

**Pull request merged**